packages/api/lib/rulesHelpers.js

```diff
@@ -104,6 +104,44 @@ function lookupCollectionInEvent(eventObject) {
   });
 }
 
+/*
+ * Build payload from rule for lambda invocation
+ *
+ * @param {ApiRule} rule              - API rule
```

Comment:
Where is this type defined?

Reply:
It's not. All of the types in this file point to ApiRule but should be the API's actual Rule Type. 
https://github.com/nasa/cumulus/pull/3149/commits/d253a57e275ecd9babd588a95a139a3f195e7f44